src/c++/perf_analyzer/test_request_rate_manager.cc

```diff
@@ -322,16 +364,48 @@ TEST_CASE("request_rate_multiple") {
 /// Check that the inference requests for sequences
 /// follow all rules and parameters
 ///
-TEST_CASE("request_rate_sequence") {
+TEST_CASE("request_rate_sequence")
+{
   TestRequestRateManager trrm{};
   PerfAnalyzerParameters params;
-  /// TODO: Params to test/tweak:
-  /// num_of_sequences
-  /// seq_length
-  /// start_seq_id
-  /// seq_id_range
 
+  // Generally we want short sequences for testing
+  // so we can hit the corner cases more often
+  //
   params.sequence_length = 3;
+
+  // TODO: Params to test/tweak:
+  // num_of_sequences
+  //    number of active sequences at once
+  // seq_length
+  //    Random length within GetRandomSequenceLength()
+  // start_sequence_id - Where the numbering should start. Default=1
+  //    Used by loadmanager::initnewsequence
+  // sequence_id_range
+  //    This is how many sequence ids there are (?)
+
+  SUBCASE("Normal") {}
```

Comment:
Would it make sense to nest some of these subcases to generate more mixing and matching of situations for better testing coverage?

Reply:
We discussed offline and determined that permutations of test cases could be created via nested loops and then calling subclass from within the nesting. But we'll hold off since this adds extensive runtime to the unit tests.